src/app/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import GridImage from '../../components/GridImage/GridImage';
 import styles from './LandingPage.module.css';
 
 function LandingPage(): JSX.Element {
+  const history = useHistory();
+  setTimeout(() => {
+    history.push('/home');
+  }, 4000);
```

Comment:
This works well, but there could be an issue if the component re-renders. You start a new timeout on each render, thus, make sure to only start a single timeout.
This is possible with a `useEffect`:

```suggestion
  useEffect(() => {
    setTimeout(() => {
      history.push('/home');
    }, 4000);
  }, []);
```

Reply:
Thanks, I didn't think about the re-rendering.  Changed the code now.